pkg/security/module/server.go

```diff
@@ -206,8 +207,8 @@ func (a *APIServer) GetConfig(ctx context.Context, params *api.GetConfigParams)
 // SendEvent forwards events sent by the runtime security module to Datadog
 func (a *APIServer) SendEvent(rule *rules.Rule, event Event, extTagsCb func() []string) {
 	agentContext := &AgentContext{
-		RuleID:      rule.Definition.ID,
-		RuleVersion: rule.Definition.Version,
+		RuleID:  rule.Definition.ID,
+		Version: version.AgentVersion,
```

Comment:
Should we call it AgentVersion ? Just to be sure we won't confuse it with the policy version